### PR TITLE
Decouple the issue-template code from comment_tab.tmpl 

### PR DESCRIPTION
**Pull request by wxiaoguang**
It would help  #23290

The issue-template code is only useful for "new issue" or "new PR", so it could only be put in the `new_form.tmpl`

@wolfogre how do you think?

### CODE REVIEW

File: templates/repo/issue/comment_tab.tmpl
Comment (delvh) [inline templates/repo/issue/comment_tab.tmpl:15]: Alternatively, we could also extract the lines below into its own subtemplate, so that we don't need to declare them twice.

**Review by delvh — COMMENTED** [review]
Logic seems to be the same.
Apart from the comment below, I see neither something that would hold this PR back, nor how that would be helpful for the issue you linked

File: templates/repo/issue/comment_tab.tmpl
Comment (wxiaoguang) [reply templates/repo/issue/comment_tab.tmpl:15]: My plan is introducing a "ComboEditor" for Gitea in the future, a ComboEditor could have: markdown toolbar, preview, textarea, dropzone(uploader).

So leave it to future.

**Comment by wxiaoguang** [comment]
> nor how that would be helpful for the issue you linked

It  would make introducing the new editor easier. Otherwise if a lot of logics are mixed together, it's difficult to get clear refactoring. In my plan, the first step is making the "comment_tab.tmpl" use the new "ComboEditor", the "issue template" part is totally unrelated.

**Review by wolfogre — APPROVED** [review]
Since it follows the same logic, I would say yes if it could benefit another PR.

**Review by lunny — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by jolheiser** [comment]
🎺 🤖

**Pull request merged**
